storybook/stories/Button/IconButton.stories.js

```diff
@@ -1,15 +1,64 @@
 import React from 'react';
+import { Text, Linking } from 'react-native';
 
 import { storiesOf } from '@storybook/react-native';
 import { action } from '@storybook/addon-actions';
 
 import IconButton from './IconButton';
 import CenterView from '../CenterView';
 
-storiesOf('Button', module)
-	.addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-	.add('IconButton', () => (
-		<React.Fragment>
-			<IconButton />
-		</React.Fragment>
-	));
+storiesOf('IconButton', module)
```

Comment:
Awesome! Make a snapshot for all your stories too

Reply:
Gotchya, I'll add them in.